blogs/2023/11/13/vscode-copilot-smarter.md

```diff
@@ -0,0 +1,175 @@
+
+![Source Control input box with sparkle to generate commit message](generate-commit-message.png)
+
+>I'm so excited about this experience, I even made this commit the other day when editing some Markdown using vscode.dev/github because the Copilot extension isn't yet [web enabled](https://code.visualstudio.com/api/extension-guides/web-extensions).
+
+>![Commit message wishing that Copilot worked in VS Code for the Web](wish-copilot-web.png)
+
+Back to our flow, let's keep going and make a pull request. I have the [GitHub Pull Request and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) extension installed, which is aware of the presence of the Copilot Chat extension. When I make a PR, there is another sparkle icon next to the title and description. Select it and Copilot writes a nice description automatically!
+
+![GitHub Pull Request and Issue extension Create view with sparkle to generate title and description](generate-pr-title-description.png)
+
+This is another area where AI can help you be more productive by automatically taking care of the simple yet tedious tasks you do 10s or 100s of times a week.
+
+## Say what?
+
+And finally, making a truly smart AI means making it as effortless as possible to interact with.
+
+We've gotten very good as an industry at voice recognition technology over the past few years. We know a lot of folks have been longing for the combination of voice assistants with advanced LLMs. Now, you can use the two together in VS Code.
+
+The new [VS Code Speech](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech) extension brings voice to text support to VS Code. Once installed, you'll see a microphone icon in all the natural language input dialogs. Select it, ask Copilot your question, and enjoy the magic.
+
+![Copilot chat input box with Speech extension microphone button](speech-extension-microphone.png)
+
+How do ya like them apples!? I told you my boy's wicked smart!
+
+![Copilot chat answer to best code editor for node and express app](best-editor-answer.png)
+
+The extension is still in preview and only supports English right now, but we'll continue to update it with new languages and capabilities over the coming months.
+
+## Work smarter, not harder
+
+All of the above and more is available in VS Code today! You just have to install the [GitHub Copilot]( https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension. And you can learn more about Copilot Chat features in [GitHub Copilot in VS Code](https://code.visualstudio.com/docs/editor/github-copilot#_chat-features)
+
+Thanks!
+
+Chris, Isi, and the VS Code team
```

Comment:
Let's just say "Chris and the VS Code team", Kai had a good point that the blog is written mainly in the first person, so there should be one person that is signed under the blog post. And Chris is a better fit due to the Boston background of the blog